style.css

```diff
@@ -0,0 +1,47 @@
+h1 {
+  font-size: 200px;
+  color: white;
+ 
+  
+}
+
+h2 {
+  color: white;
+  text-align: left;
```

Comment:
This reiterates the  [default browser styles](https://browserdefaultstyles.com/) for headings and, therefore, can be removed:
```suggestion
```